### PR TITLE
WIP - Resource manager tests

### PR DESCRIPTION
**Pull request by tveverka-RH**
# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

File: tests/utils/test_resource_manager.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""
+Copyright (c) 2022 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import pytest
+from unittest import mock
```
Comment (MartinBasti) [inline tests/utils/test_resource_manager.py:9]: flexmock?

File: tests/utils/test_resource_manager.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+from atomic_reactor.utils.resource_manager import Host, ResourceManager
+
+
+def test_host_init():
```
Comment (MartinBasti) [inline tests/utils/test_resource_manager.py:13]: I'd suggest to add into classes here

```
class TestHost:

  def test_host_init(self):
 ...
  def test_host_read_info_data(self)
...
```

Comment (tveverka-RH) [reply tests/utils/test_resource_manager.py]: Fixed

File: tests/utils/test_resource_manager.py (hunk 1 above)
Comment (tveverka-RH) [reply tests/utils/test_resource_manager.py:9]: is it issue? I'm not so familiar with it - I'm trying to change it to fm but it's not so intuitive I would say

**Pull request closed without merging**